eks/modules.tf

```diff
@@ -1,13 +1,11 @@
 module "network" {
   source = "./modules/network"
-
-  cluster-name    = "${var.cluster-name}"
 }
 
 module "eks" {
     source = "./modules/eks"
 
-    cluster-name  = "${var.cluster-name}"
```

Comment:
I preferred leaving this around, it allowed to easily see which clusters you own on EKS, and which to delete.

Reply:
> I preferred leaving this around, it allowed to easily see which clusters you own on EKS, and which to delete.

It's better for CI to randomize the cluster names. You should still be able to see from terraform output what cluster it is, admittedly not a very mnemonic-friendly name. It starts with cap- and postfixed with 12 random characters.